webservices_dispatch_action/rerendering.py

```diff
@@ -80,11 +80,22 @@ def comment_and_push_per_changed(
                 "https://conda-forge.org/docs/maintainer/updating_pkgs.html"
                 "#rerendering-with-conda-smithy-locally"
             )
+            global_pinning_url = (
+                "https://github.com/conda-forge/conda-forge-pinning-feedstock/"
+                "blob/master/recipe/conda_build_config.yaml"
+            )
             message = """\
 Hi! This is the friendly automated conda-forge-webservice.
 I tried to rerender for you but ran into some issues, please ping conda-forge/core
 for further assistance. You can also try [re-rendering locally]({}).
-""".format(doc_url)
+
+More information on the issue may be available in the actions tab of the feedstock
+under a "rerender" workflow. The following suggestions might explain the problem:
```

Comment:
I don't really know what a rerender workflow is... one of the bot magic spells?

Reply:
If you look on the "actions" tab of the feedstock: https://github.com/conda-forge/geckodriver-feedstock/actions

"workflow" is then the name for the pipeline/"type of CI job" in GitHub actions lingo. Any ideas for how to phrase it more clearly?

Reply:
Ah, ok... so it's not a workflow i, the maintainer, requested, but a bot one.

The cadillac would be to generate a link to the workflow in question, as even the failed workflow comes up green... but it should be the last one. Failing that, is the current feedstock in scope for this message?

```
https://github.com/conda-forge/{{ feedstock }}/actions?query=workflow%3A.github%2Fworkflows%2Fwebservices.yml
```

Reply:
the workflow link should be reconstructable given the variables github puts in the environment.